### PR TITLE
docs(admin-api) add examples using arrays

### PR DESCRIPTION
**Pull request by Tieske**
most pressing issue: example of array usage on the command line

### CODE REVIEW

File: autodoc/data/admin-api.lua [hunk 1]
```diff
@@ -78,6 +78,17 @@ return {
         config.limit=10&config.period=seconds
         ```
 
+        When specifying arrays use square brackets (numbering inside the brackets is optional
```
Comment (bungle) [inline autodoc/data/admin-api.lua:81]: Or just send them in order
```
-d "paths=/path/one" \
-d "paths=/path/two"
```

Comment (bungle) [reply autodoc/data/admin-api.lua:81]: This also applies to `multipart/form-data` but there is a bug in `multipart` library where it does not allow same parameter multiple times (it overwrites previous ones), and thus with multipart you really need to use indexes `paths[1]`, `paths[2]`.

Comment (Tieske) [reply autodoc/data/admin-api.lua:81]: multi-part was removed from the docs. Hence didn't reference it here anymore either

Comment (bungle) [reply autodoc/data/admin-api.lua:81]: multipart is rather useful when uploading certs.

Comment (bungle) [reply autodoc/data/admin-api.lua]: and multipart is not removed, it has been manually added: https://github.com/Kong/docs.konghq.com/commit/a54c86139d732034fd1d1c70b0b2df395f87bff1#diff-27c036e2fa20a4521cbd01188afdc4f0

Comment (Tieske) [reply autodoc/data/admin-api.lua]: I know, but why was it removed here then?

Comment (bungle) [reply autodoc/data/admin-api.lua]: I think it has never been here.

Comment (bungle) [reply autodoc/data/admin-api.lua]: I checked, and it has never been.

**Pull request merged**